source/api/presentation/3.0/index.md

```diff
@@ -524,19 +524,19 @@ The value _MUST_ be a string, taken from the table below or a full URI.
 
 #### 3.4.1 External Links
 
-##### related
-A link to an external resource that describes or is about this resource. The external resource _MUST_ be able to be displayed directly to the user. Examples might include a video or academic paper about the resource, a website, an HTML description, and so forth.
+##### homepage
+A link to an external web page that primarily describes the real world object that this resource is a presentation of. The web page is usually published by the organization responsible for the real world object, and might be generated by a collection management system or hand crafted. The external resource _MUST_ be able to be displayed directly to the user.  External resources that are related, but not home pages, _MUST_ instead be added into the `metadata` property, with an appropriate `label` or `value` to describe the relationship.
```

Comment:
> is a presentation of

represents

>  collection management system or hand crafted.

collection management system or library catalog. ? Hand-crafted seems odd to me.